<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Changan</class>
 <widget class="QDialog" name="Changan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>728</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1071</width>
    <height>728</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1071</width>
    <height>728</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>无涯</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/image/Resources/无涯.png</normaloff>:/image/Resources/无涯.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/image/Resources/changan.png);</string>
  </property>
  <widget class="QPushButton" name="btnKezhan">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
font: 22pt &quot;隶书&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>客栈</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnYiguan">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>360</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 255, 255);
font: 24pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>医馆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMinju2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 255, 255);
font: 16pt &quot;隶书&quot;;
</string>
   </property>
   <property name="text">
    <string>民居</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMinju1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 255, 255);
font: 30pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>民居</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>540</y>
     <width>921</width>
     <height>171</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/Resources/透明背景.png);
color: rgb(255, 255, 255);
font: 16pt &quot;隶书&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOut">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 255, 255);
font: 14pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>出城</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Menu.qrc"/>
 </resources>
 <connections/>
</ui>
